holoviews/core/util.py

```diff
@@ -1274,6 +1269,10 @@ def is_dataframe(data):
     """
     Checks whether the supplied data is of DataFrame type.
     """
+    if 'dask' in sys.modules:
+        import dask.dataframe as dd
+    else:
+        dd = None
```

Comment:
Could also do:

```
dd = None
if 'dask' in sys.modules:
   import dask.dataframe as dd
```

Saves a line and maybe even more explicit that ``dd` always exists...

Reply:
This isn't a strong opinion but I do think my suggested version is slightly better. Don't you?

Reply:
Basically have no preference here.